app/src/androidTest/java/org/mozilla/fenix/ui/HomeScreenTest.kt

```diff
@@ -84,11 +56,12 @@ class HomeScreenTest {
 
             verifyWelcomeHeader()
 
-            verifyAlreadyHaveAnAccountHeader()
             verifyGetTheMostHeader()
             verifyAccountsSignInButton()
             verifyGetToKnowHeader()
 
+            swipeUp()
```

Comment:
Is this an okay way to deal with the recycler view content? When I do "scroll to" it doesn't work for elements that are off screen.

Reply:
you probably tried this, but just in case, how about `scrollToPositionWithOffset`? would it work? https://stackoverflow.com/questions/26875061/scroll-recyclerview-to-show-selected-item-on-top

Reply:
I mean, swipeup is fine, it's also fun to try what happens with above

Reply:
> Is this an okay way to deal with the recycler view content? When I do "scroll to" it doesn't work for elements that are off screen.

You can also do something like this:
```
    fun scrollToElementByText(textString: String): UiScrollable {
        mDevice.wait(Until.findObject(By.text(textString)), TestAssetHelper.waitingTime)
        val matchedScrollableElement = mDevice.findObject(
            UiSelector()
                .text(textString)
        )
        val appView = UiScrollable(UiSelector().scrollable(true))
        appView.scrollIntoView(matchedScrollableElement)
        return appView
    }
```

Reply:
In the test itself, it usually better to give more user-centric names so it's clear what the intent is.  For example, here you might want to say:  swipeUpToDismissFirstRun.  Then the tests don't need explanatory comments.